zk_toolbox/crates/zk_inception/src/commands/ecosystem/init.rs

```diff
@@ -119,6 +119,8 @@ pub async fn run(args: EcosystemInitArgs, shell: &Shell) -> anyhow::Result<()> {
             genesis_args: genesis_args.clone().fill_values_with_prompt(&chain_config),
             deploy_paymaster: final_ecosystem_args.deploy_paymaster,
             l1_rpc_url: final_ecosystem_args.ecosystem.l1_rpc_url.clone(),
+            // NOTE: Consider adding port offset to the multiple chain configs
+            port_offset: 0,
```

Comment:
we have id in our config, probably we could reuse id instead of port offset

Reply:
Addressed in https://github.com/matter-labs/zksync-era/pull/2594/commits/b83961ccaac89f857430fa888830ef30c1c01479